src/Nut/DebugServiceRegistration.php

```diff
@@ -22,6 +25,7 @@ protected function configure()
         $this
             ->setName('debug:services')
             ->setDescription('System service provider registration boot order debug dumper.')
```

Comment:
Can we change the name to `debug:providers` or `debug:service-providers`? Individual services are not listed here. Also update the class name to match. 

Description: Dumps service providers and their order.